core/src/App.html

```diff
@@ -1192,6 +1151,21 @@
             },
             {}
           );
+        }else {
+          let userSettingsIframe = UserSettingsHelper.findActiveCustomUserSettingsIframe(e.source);
+          if(userSettingsIframe){
+            let userSettingsGroupKey = userSettingsIframe.getAttribute('userSettingsGroup');
+          let config = {
+            context: {
+              userSettingsData: storedUserSettings[userSettingsGroupKey]
+            }, // nur für spezielen Iframe 
+            iframe: 
+              userSettingsIframe
+              
+          }
```

Comment:
```suggestion
            let userSettingsGroupKey = userSettingsIframe.getAttribute('userSettingsGroup');
            let config = {
              context: {
                userSettingsData: storedUserSettings[userSettingsGroupKey]
              }, // nur für spezielen Iframe 
              iframe: userSettingsIframe
          }
```
Just stupid indentation ....